tests/unit/openssl_lib_mock.h

```diff
@@ -35,6 +35,13 @@ namespace openssl
 class OpenSSLLibMockInterface
 {
 public:
+    virtual const char* SSL_EC_curve_nid2nist(int nid) = 0;
```

Comment:
Where are the actual tests that use the mock? I would expect them to be added in MoCOCrW/tests/unit/test_hsm.cpp

Reply:
All code paths are covered by the examples. That's why I initially decided not to write unit tests, especially since example tests the functionality E2E and has more chance of reporting a problem.

Reply:
Added some unit tests to test_opensslwrapper.cpp